socnet/templates/blog/subscribers.html

```
@@ -7,28 +7,32 @@
       <a
-        href="{{ subscriber.get_absolute_url }}"
         class="list-group-item list-group-item-action d-flex align-items-center"
+        href="{{ subscriber.get_absolute_url }}"
       >
         <div
           class="avatar-thumbnail d-flex align-items-center justify-content-center me-2"
         >
           {% if subscriber.image %}
-            <img src="{{ subscriber.image.url }}" loading="lazy" class="rounded" />
+            <img
+              class="rounded"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)